front/components/assistant/conversation/input_bar/editor/useHandleMentions.tsx

```diff
@@ -62,12 +52,33 @@ const useHandleMentions = (
           editorService.getTrimmedText() ?? null;
       }
     }
-  }, [
-    agentConfigurations,
-    editorService,
-    mentionedAgentConfigurationIds,
-    disableAutoFocus,
-  ]);
+  }, [agentConfigurations, editorService, stickyMentions, disableAutoFocus]);
+
+  useEffect(() => {
+    if (selectedAssistant) {
+      const agentConfiguration = agentConfigurations.find(
+        (agent) => agent.sId === selectedAssistant.configurationId
+      );
+
+      if (!agentConfiguration) {
+        return;
+      }
+
+      const { mentions: currentMentions } = editorService.getTextAndMentions();
+      const hasMention = currentMentions.some(
+        (mention) => mention.id === agentConfiguration.sId
+      );
```

Comment:
Can you move this to the editor service, it's okay to have a method there like `hasMentions()`.

Reply:
will do in my next PR (in review in 10mns)